test/index-test.js

```diff
@@ -1,15 +0,0 @@
-/* eslint-disable no-unused-expressions */
-import {expect} from 'chai';
-import * as geoapi from '../src/index.js';
-
-describe('index', () => {
-  it('should export a geocoding interface', () => {
-    expect(geoapi.Geocoding).to.be.a('function');
-  });
-  it('should export a static maps interface', () => {
-    expect(geoapi.staticMaps).to.exist;
```

Comment:
Fair point. It's a semantics issue I suppose. I'm not sure which one I prefer. It might be confusing two have two different endpoints for the same API. On the other hand it might be confusing to have two endpoints within an endpoint. I think splitting it up might be more appropriate.